src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -242,6 +242,29 @@ public void MaxRequestContentBufferSize_SetInvalidValue_ThrowsArgumentOutOfRange
             }
         }
 
+        [Theory, MemberData(nameof(RedirectStatusCodes))]
```

Comment:
This test needs to be Outerloop especially since it uses an external test endpoint.

Reply:
Will do.